src/components/Root/Root.js

```diff
@@ -97,10 +97,16 @@ class Root extends Component {
       setToken: (val) => { store.dispatch(setOkapiToken(val)); },
       actionNames,
       locale,
+      timezone,
       metadata,
-      setLocale: (val) => { loadTranslations(store, val); },
+      setLocale: (localeValue) => { loadTranslations(store, localeValue); },
+      setTimezone: (timezoneValue) => { store.dispatch(setTimezone(timezoneValue)); },
       plugins: plugins || {},
       setSinglePlugin: (key, value) => { store.dispatch(setSinglePlugin(key, value)); },
+      getDateTime: (dateStr, zone) => getDateTime(dateStr, zone || timezone),
```

Comment:
It's not clear to me that these non-Stripes-related utility functions belong on the Stripes object. Is there a reason for this?

Reply:
@MikeTaylor We have `formattedDate` and `formattedTime` utilized all over the apps, in stripes-* and ui-*. So, extending the stripes object provides a centralized way in which these functions could be accessed, instead of importing it from the `react-intl` everytime. With this, we can remove a lot of `util` files that are present in every ui-* app that primarily implement these 2 functions.

Reply:
This doesn't sound right to me. The Stripes object has a specific function -- i.e. supplying Stripes-specific functionality to client code. It's not just a handy place to hang any old bit of functionality that we can't be bothered to import.

If
```
return this.props.stripes.getTImeDate(dateStr, timezone);
```
is merely a shorthand for
```
import moment from 'moment-timezone';
return moment.tz(dateStr, timezone);
```
then I think we should say what we mean.

Reply:
And, needless to say, the `util` files, if they are used in many different apps, should not be replicated in those apps but moved to a central place whence they can be shared.

Reply:
Oh, my bad. I didn't realize you were talking about the `getDateTime()` function. Yeah, that can definitely be moved out of stripes. Will do it.

Reply:
Well, I'm not really clear on what any of them do. Let me ask you this: what code to you have to write at present to use these other functions, and what would you be able to reduce it to if these functions came on the Stripes object?

Reply:
So, the `getDateTime()` function is not being used anywhere else in the app. Its just being used in the `dateUtil` file which I have added with this PR, and I agree thats redundant and can be removed. I will remove it now.

As far as the other functions:

Code that is currently present:

in every app you have a `util.js` file that contains 2 functions: `formatDate` and `formatTime`
With this PR it is now just present in one file on stripes-core

Also, in each app:

for eg: https://github.com/folio-org/ui-users/blob/master/lib/Loans/ClosedLoans/ClosedLoans.js#L6. 

You have such lines in every app, that has the `util.js` file with the same function body.

Also, the `timezone` string that is now present through stripes object needn't be passed around all over the apps ( because you would have to then pass the timezone as an argument/ and pass it a prop to the `FormattedDate` and `FormattedTime` in every function that resides in all the `util` files). Now, with this PR, It just resides in stripes-core based on which the date and time can be formatted.

So, with the current PR, it comes down to: (just one line)

`props.stripes.formatDate(dateStr)`

With the already present code: 

=> import { formatDate } from '../util'
=> timezone = props.stripes.timezone;
=> `formatDate(dateStr, timezone)` 

and in the util file (this is in every app)
=> import { formattedDate } from 'react-intl'
=>  formatDate = (dateStr, timezone)) => `<formattedDate value={dateStr} timezone={timezone} />

Reply:
OK, so there are two things going on here.

First, it's clearly idiotic that every app is carrying its own mostly-redundant-but-slightly-divergent `util.js` (or in some cases, I notice, `utils.js`, plural). Obviously such code should by DRYed and put in a shared location.

In general, the Stripes object is _not_ that location: such functions should usually go in `@folio/stripes-components/util' -- there's plenty of precedent. (One day, we may separate that area out into its own `@folio/stripes-util`, but it is not this day.)

_However_, in the present case, I see that `stripes.formatDate` is currying the underlying function with a value that is known to Stripes -- that is, the timezone. I do agree that this changes the equation. What this means is that what we have here really is a _Stripes-aware_ date-formatting function -- that is, aware of the timezone that Stripes has established. So, yes, that one does belong on the Stripes object.

I've not looked at the others, but you can make your own judgement based on these principles.

Sorry this has dragged on a bit. I don't like to be That Guy, but I hope it's been productive for you.

Reply:
Sure, its been informative. Will remove the `getDateTime()` function and merge it in a bit. Thanks

Reply:
Thank you!